frontend/public/components/catalog-items.jsx

```diff
@@ -225,6 +226,10 @@ export class CatalogTileViewPage extends React.Component {
     this.syncTabsAndTiles(category, parent);
   }
 
+  onItemClick(href) {
+    history.push(href);
+  }
```

Comment:
I'd suggest

```jsx
onItemClick = href => {
  history.push(href);
}
// ...
onClick={this.onItemClick}
```

This avoids creating a new anonymous function on every render.

You might even be able to do

```jsx
onClick={history.push}
```

Reply:
The `onltemClick` is outside of the render method and is on the Class.
I left it as a separate function since I know we are looking to potentially change the click from navigating to showing a side panel.

Reply:
Right, I understand. You can use arrow functions as class members. Here is an example:

https://github.com/openshift/console/blob/master/frontend/public/components/source-to-image.tsx#L113

I'd suggest either doing that or binding to this in the constructor.

Reply:
Gotcha, updated.

Reply:
Thanks, did you push the changes? I don't see them.

Reply:
https://github.com/openshift/console/pull/665/commits/c502d9686cccc5e603c957df4ba07bead1d14ae2#diff-c1ded5dc8e6607f6aa5ef01f9d493dabR57

Reply:
OK, thanks. I wonder if it only works in TypeScript files. Hm.